helper/snapshot/snapshot_test.go

```diff
@@ -110,7 +110,7 @@ func makeRaft(t *testing.T, dir string) (*raft.Raft, *MockFSM) {
 
 	timeout := time.After(10 * time.Second)
 	for {
-		if raft.Leader() != "" {
+		if leaderID, _ := raft.LeaderWithID(); leaderID != "" {
```

Comment:
```suggestion
		if leaderAddr, _ := raft.LeaderWithID(); leaderAddr != "" {
```

ID is the second return value.